apps/backend/src/deposits/entities/pos-deposit.entity.ts

```diff
@@ -43,12 +52,23 @@ export class POSDepositEntity {
   @Column({ default: false })
   match: boolean;
 
+  // rename this to just payment id?
   @Column({ nullable: true })
   matched_payment_id?: string;
 
-  method?: number;
+  @ManyToOne(() => PaymentMethodEntity, (pd) => pd.method)
```

Comment:
We need these join columns to properly get the entities with the relationships, including lookups. 

This makes querying so much easier and readable.

Reply:
This is only a typeorm config and does not create a column